src/sensai/torch/torch_models.py

```diff
@@ -148,8 +147,8 @@ def _predictOutputsForInputDataFrame(self, inputs: pd.DataFrame) -> np.ndarray:
         result = super()._predictOutputsForInputDataFrame(inputs)
         return np.squeeze(result, 2)
 
-    def _computeInputs(self, x: pd.DataFrame, y=None, fit=False) -> pd.DataFrame:
-        x = super()._computeInputs(x, Y=y, fit=fit)
+    def computeProcessedInputs(self, x: pd.DataFrame, Y: pd.DataFrame = None, fit=False) -> pd.DataFrame:
+        x = super().computeProcessedInputs(x, Y=Y, fit=fit)
```

Comment:
I am a little bit unsure whether this change is correct (like 90% sure it's fine, but still). It might deserve a critical look from you